src/schemas/projects.ts

```diff
@@ -3,10 +3,19 @@ import Joi from 'joi';
 export const projectInputSchema = Joi.object({
   name: Joi.string().required(),
   date: Joi.date().required(),
-  description: Joi.string().required(),
+  sections: Joi.array()
+    .items(
+      Joi.object({
+        title: Joi.string().optional(),
+        body: Joi.string().optional(),
+        imageUrl: Joi.array().items(Joi.string()).optional(),
```

Comment:
imageURL

Reply:
I have made the necessary changes requested